libyul/optimiser/DataFlowAnalyzer.h

```diff
@@ -85,6 +86,8 @@ class DataFlowAnalyzer: public ASTModifier
 		std::set<YulString> variables;
 		bool isFunction;
 	};
+	/// Special expression whose address will be used in m_value.
+	Expression const m_zero{Literal{{}, LiteralKind::Number, YulString{"0"}, {}}};
```

Comment:
How are yul string references held by yulstringrepo due to class members such as these reset?

Reply:
The idea is that the DataFlowAnalyzer is a short-lived object that is destroyed again early enough.